### ISSUE TITLE
Need to add a Dockerfile to `v2.0-dev` to run CI

### ISSUE THREAD

**Issue by freemansw1** [climate-processes/tobac#69]
Now that I've added v2.0-dev to our Github Actions CI, I think we need to populate both `main` and `v2.0-dev` with a Dockerfile to run. `main` will be easy, I will just PR `dev` to `main`, but for `v2.0-dev`, I'll need to do a separate PR, probably after #65 is merged.

**Comment by freemansw1** [comment]
This will be resolved by #73
